editor/filesystem_dock.cpp

```diff
@@ -1595,6 +1595,11 @@ void FileSystemDock::_update_dependencies_after_move(const HashMap<String, Strin
 	// The following code assumes that the following holds:
 	// 1) EditorFileSystem contains the old paths/folder structure from before the rename/move.
 	// 2) ResourceLoader can use the new paths without needing to call rescan.
+
+	// The currently edited scene should be reloaded first, so get it's path (GH82652).
+	const EditorData &ed = EditorNode::get_singleton()->get_editor_data();
+	const uint64_t edited_scene_path_hash = ed.get_scene_path(ed.get_edited_scene()).hash64();
```

Comment:
This can be simplified to
```suggestion
	const uint64_t edited_scene_path_hash = EditorNode::get_editor_data().get_scene_path(ed.get_edited_scene()).hash64();
```

Reply:
There's two uses of `ed` in that line so not that trivial

Reply:
Ah ok then. Still, the `get_singleton()` part is not needed, because `get_editor_data()` is static.